builtin/game/fine_pointed.lua

```diff
@@ -0,0 +1,35 @@
+-- Returns the exact coordinate of a pointed surface
+
+local function fine_pointed(pos, camera_pos, pos_off, look_dir)
+	local offset, nc
+	local oc = {}
+	for c, v in pairs(pos_off) do
+		if v == 0 then
+			oc[#oc + 1] = c
+		else
+			offset = v
+			nc = c
+		end
+	end
+
+	local fine_pos = {[nc] = pos[nc] + offset}
+
+	camera_pos.y = camera_pos.y + 1.625
```

Comment:
This was tough. I found [this](https://github.com/minetest-australopithecus/minetest-australopithecus-sneak-cam) mod, which lowers the camera position while sneaking.

I have to divide the offset by 10, however